### PR TITLE
Update node-forge

### PR DESCRIPTION
**Pull request by killalau**
Please fill in this template.
- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] The package does not provide its own types, and you can not add them.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If changing an existing definition:
- [x] Provide a URL to  documentation or source code which provides context for the suggested changes: <https://github.com/digitalbazaar/forge>
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by westy92** [comment]
LGTM! Please add more "tests" to cover this new functionality.

**Comment by killalau** [comment]
I've add test case, not sure if this look good. It my first time to contribute DefinitelyTyped

**Comment by westy92** [comment]
Better!  Ideally we'd have usage of each new method and type in the tests file.  You can typically base them on examples or tests from the project repository.
https://github.com/digitalbazaar/forge

As a side note, I'm not sure if PRs are supposed to go directly to the types-2.0 branch.  I believe the DT team wants PRs to go to master and they occasionally merge master back up to types-2.0.  I am not positive on that.

Thank you for your contribution!

**Comment by killalau** [comment]
Thanks a lot.

When I go through the readme,
https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request
and
https://github.com/DefinitelyTyped/DefinitelyTyped#faq

types-2.0 branch will auto publish to npm `@types` package. I think this is the new practise.

**Comment by killalau** [comment]
Do you have permission to edit the `dt-bot` comments? I guess it is a form you should fill in order to approve the PR (actually i am not so sure the workflow here, things so automatically like a magic)

**Comment by westy92** [comment]
I've approved it.

File: node-forge/node-forge-tests.ts [hunk 1]
```diff
@@ -1,7 +1,80 @@
+/// <reference path="./index.d.ts" />
```
Comment (killalau) [inline node-forge/node-forge-tests.ts]: You are write, i forgot to remove it

**Pull request merged**